homepage/views.py

```diff
@@ -9,7 +9,10 @@ def app_layout(request):
 
 
 def landing(request):
-    return render(request, 'homepage/landing/landing.html')
+    last_reviews = Review.objects.all().order_by('-date')
```

Comment:
```suggestion
    last_reviews = Review.objects.all().order_by('-date')[:3]
```

Reply:
Great, I thought [:3] will raise an error when theres less than 3, but python keeps surprising me :)
Changed and moved this logic to a new `landing_page_feed()` function in order to test is separately.